src/Illuminate/Testing/TestResponse.php

```diff
@@ -692,13 +692,13 @@ public function assertJsonMissingValidationErrors($keys = null, $responseKey = '
 
         $json = $this->json();
 
-        if (! array_key_exists($responseKey, $json)) {
-            PHPUnit::assertArrayNotHasKey($responseKey, $json);
+        if (! Arr::has($json, $responseKey)) {
+            PHPUnit::assertFalse(Arr::has($json, $responseKey), 'Response has unexpected '.$responseKey.' key.');
```

Comment:
This looks confusing to me honestly. I would've simplified this with the following:
```suggestion
            PHPUnit::assertTrue(true);
```